wic/src/main/java/edu/asu/diging/wic/config/SecurityContext.java

```diff
@@ -39,7 +39,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 // Anyone can access the urls
                 .antMatchers("/", "/network", "/concept/**", "/resources/**", "/login", "/login/authenticate",
-                        "/logout").permitAll()
+                        "/logout","/concepts/types/all").permitAll()
```

Comment:
since "concept/**` is already permitted to all, just make the url "/concept/types/all"